evm/src/test/java/org/hyperledger/besu/evm/operations/Create2OperationTest.java

```diff
@@ -302,4 +305,58 @@ private MessageFrame testMemoryFrame(
         memoryOffset.trimLeadingZeros().toInt(), SIMPLE_CREATE.size(), SIMPLE_CREATE);
     return messageFrame;
   }
+
+  @org.junit.Test
```

Comment:
Is this reference to junit4's `@Test` on purpose?

Reply:
No, it leaked in because CreateOperationTest is still on the old JUnit annotations.  Fixed as well as CreateOperationTest.